net-proxy/3proxy/3proxy-0.7.1.2.ebuild

```diff
@@ -0,0 +1,25 @@
+# Copyright 1999-2012 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Header: $
+EAPI=5
+inherit eutils
+
+DESCRIPTION="3Proxy is really tiny cross-platform proxy servers set"
+HOMEPAGE="http://www.3proxy.ru/"
+SRC_URI="http://3proxy.ru/${PVR}/${P}.tgz"
+LICENSE="GPL3"
+SLOT="0"
+KEYWORDS="~x86 ~amd64"
+IUSE=""
+RDEPEND="${DEPEND}"
+
+S="${WORKDIR}/${PN}"
+
+src_prepare() {
+        epatch "${FILESDIR}"/${PN}-${PVR}-gentoo.patch
```

Comment:
`${PN}-${PVR}` -> `${PF}`. But it's usually a very bad idea to force different patch on every revision. Some developers may need to bump the revision for some minor change, and then they'd be confused by the need to update the patch.